Source/monster.cpp

```diff
@@ -3400,7 +3426,7 @@ BOOL M_CallWalk2(int i, int md)
 	int mdtemp;
 
 	mdtemp = md;
-	ok = DirOK(i, md);     // Can we continue in the same direction
+	ok = DirOK(i, md);          // Can we continue in the same direction

```

Comment:
fix : p

Reply:
Comments are aligned and there is a comment on the next line 😕